src/tools/create-file-stasher.js

```diff
@@ -75,9 +75,12 @@ const createFileStasher = (input) => {
       return ZapierPromise.reject(new Error('rpc is not available'));
     }
 
-    const isRunningOnHydrator = _.get(input, '_zapier.event.method', '').indexOf('hydrators.') === 0;
-    if (!isRunningOnHydrator) {
-      return ZapierPromise.reject(new Error('Cannot stash files outside an hydration function/method.'));
+    const isRunningOnHydratorOrCreate = () => {
```

Comment:
This looks good! I'd suggest considering keeping `isRunningOnHydrator` and adding a `isRunningOnCreate`, then you can do the check with `if (!isRunningOnHydrator && !isRunningOnCreate)`. I think it might become more clear and easier to add a `isRunningOnSearch` later if we find it necessary. Again, no need to take the suggestion!

Reply:
Great idea - done!